### PR TITLE
Add exec-wrappers recipe

### PR DESCRIPTION
**Pull request by gqmelo**

### CODE REVIEW

File: recipes/exec-wrappers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "281a405b33bff853745f46408cc80bb3105609c12454fb71bc378dbe8e0419c4" %}
+
+package:
+  name: exec-wrappers
+  version: {{ version }}
+
+source:
+  fn: simplejson-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/exec-wrappers/meta.yaml]: Should be `exec-wrappers`, right? :wink:

**Comment by jakirkham** [comment]
Could I ask that we break out `exec-wrappers` in a separate PR? It looks like that one is basically good to go with a very minor change. That way it will be easier to release `conda-wrappers` after.

File: recipes/exec-wrappers/meta.yaml (hunk 1 above)
Comment (gqmelo) [inline recipes/exec-wrappers/meta.yaml]: Duh! Fixed, thanks!

**Comment by gqmelo** [comment]
> Could I ask that we break out exec-wrappers in a separate PR? It looks like that one is basically good to go with a very minor change. That way it will be easier to release conda-wrappers after.

Of course, done!
I was in doubt what to do when there is a recipe that depends on other. In this case, is there a way that I could build ```conda-wrappers``` on another PR before merging ```exec-wrappers```? Or should I just replicate the commit on ```conda-wrappers``` PR?

**Comment by jakirkham** [comment]
Of course. Normally I just do 2 PRs then. That way I can work on both, but get the dependencies in and converted to feedstocks first.

**Comment by jakirkham** [comment]
Thanks @gqmelo.

**Comment by jakirkham** [comment]
Should add that this will probably not be needed in this case. Feel free to get `conda-wrappers` going in a new PR. We can restart the CIs once `exec-wrappers` is built.

File: recipes/exec-wrappers/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/exec-wrappers/meta.yaml]: No worries.

**Comment by jakirkham** [comment]
So you, @gqmelo, should get an email that invite you to join conda-forge. Once accepted you will be added to a team with the same name as this recipe. Those will give you permissions on the feedstock (repo) for this recipe. Make sure when proposing any change that you go through the typical GitHub workflow of forking the feedstock and making changes in your fork that you PR back. Once merged CIs will build and deploy any changes you make. Please let us know if you have any questions and welcome to conda-forge.

**Pull request merged**